cart/views.py

```diff
@@ -9,15 +9,58 @@
 from product.models import Product
 from voucher.models import Voucher
 
-
-class UserCartView(View):
-    @login_decorator
+class OrderDetailView(View):
     def post(self, request):
-        data       = json.loads(request.body)
-        user_id    = request.user.id
-
-        Cart.objects.create(
-                product_id = 
-                user_id    = user_id,
-                voucher_id = voucher_id
-                )
+        try:
+            data                        = json.loads(request.body)
+            email                       = data['email']
+            billing_country             = data['billing_country']
+            billing_first_name          = data['billing_first_name']
+            billing_last_name           = data['billing_last_name']
+            billing_street_number       = data['billing_street_number']
+            billing_additional_address  = data['billing_additional_address']
+            billing_district            = data['billing_district']
+            billing_city                = data['billing_city']
+            billing_postal_code         = data['billing_postal_code']
+            billing_phone_number        = data['billing_phone_number']
+            shipping_country            = data['shipping_country']
+            shipping_first_name         = data['shipping_first_name']
+            shipping_last_name          = data['shipping_last_name']
+            shipping_street_number      = data['shipping_street_number']
+            shipping_additional_address = data['shipping_additional_address']
+            shipping_district           = data['shipping_district']
+            shipping_city               = data['shipping_city']
+            shipping_postal_code        = data['shipping_postal_code']
+            shipping_phone_number       = data['shipping_phone_number']
+            
+            if User.objects.filter(email=email).exists():
+                user = User.objects.get(email=email)
+            else:
+                user = User.objects.create(email=email, is_anonymous=True)
+            
+            AddressInformation.objects.create(
+                user_id=user.id,
+                billing_country=billing_country,
+                billing_first_name=billing_first_name,
+                billing_last_name=billing_last_name,
```

Comment:
```suggestion
                user_id            = user.id,
                billing_country    = data['billing_country'],
                billing_first_name = data['billing_first_name'],
                billing_last_name  = data['billing_last_name'],
```
1. =을 기준으로 라인 맞춰 주세요.
2. 프론트에서 받아오는 데이터를 변수에 담지 말고 바로 그냥 사용하셔도 될 것 같습니다. 이번 같은 경우는 변수에 담아서 여러번 사용하지 않고, 바로 데이터를 저장하기 때문에 변수에 따로 저장하지 않아도 될 것 같아요.

Reply:
위코드 코딩 컨벤션 반영하였습니다~
데이터 변수에 담은 부분 삭제했습니다~~ 감사합니다!